common/config/babel/.babelrc.js

```diff
@@ -34,7 +34,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // dialpad
         'dialpad',
         // rooms
-        'rooms'
+        'rooms',
+        // Call Readiness
```

Comment:
nit: delete the comment. It doesn't add anything that the well named flag doesn't say already.